src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -274,6 +281,9 @@ void persistInitialLaunchParams(KernelAlternatives kernelAlts) {
             logger.atDebug().log("Nucleus launch parameters has already been set up");
             return;
         }
+	//  Android does't separate java options per application
+#if ANDROID
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, delete these tokens

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.